src/main/java/com/rhombus/api/examples/FaceReport.java

```diff
@@ -0,0 +1,342 @@
+        CSVadd(csvFile, report, recentfaceData, name, cameraname, Datacamera, apiKey);
+
+    }
+
+    private static void _initialize(String apiKey)
+    {
+        /*
+         * API CLIENT
+         */
+        {
+            final List<Header> defaultHeaders = new ArrayList<>();
+            defaultHeaders.add(new BasicHeader("x-auth-scheme", "api-token"));
+            defaultHeaders.add(new BasicHeader("x-auth-apikey", apiKey));
+
+            final ClientBuilder clientBuilder = ClientBuilder.newBuilder();
+            clientBuilder.register(new JacksonJaxbJsonProvider().configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false).configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false));
+            ;
+
+            final HostnameVerifier hostnameVerifier = new HostnameVerifier() {
+                @Override
+                public boolean verify(String hostname, SSLSession session) {
+                    return true;
+                }
+            };
+
+            clientBuilder.hostnameVerifier(hostnameVerifier);
+
+            _apiClient = new ApiClient();
+            _apiClient.setHttpClient(clientBuilder.build());
+            _apiClient.addDefaultHeader("x-auth-scheme", "api-token");
+            _apiClient.addDefaultHeader("x-auth-apikey", apiKey);
+
+            _facewebservice = new FaceWebserviceApi(_apiClient);
+            _camerawebservice = new CameraWebserviceApi(_apiClient);
+        }
+        /*
+         * MEDIA/VIDEO CLIENT
+         */
+        {
+            final HttpClientBuilder httpClientBuilder = HttpClients.custom();
+
+            final HostnameVerifier hostnameVerifier = new HostnameVerifier()
+            {
+
+                @Override
+                public boolean verify(String hostname, SSLSession session)
+                {
+                    return true;
+                }
+            };
+
+            _videoClient = httpClientBuilder.setSSLHostnameVerifier(hostnameVerifier).build();
+        }
+    }
+
+    private static void savingIMG(String apikey, String thumnail, OutputStream outputStream) throws Exception
+    {
+        {
+            final HttpGet pictureRequest = new HttpGet(thumnail);
+            pictureRequest.setHeader("x-auth-scheme", "api-token");
+            pictureRequest.setHeader("x-auth-apikey", apikey);
+
+            final HttpResponse pictureResponse = _videoClient.execute(pictureRequest);
+            final byte[] VideoDataRaw = IOUtils.toByteArray(pictureResponse.getEntity().getContent());
+            IOUtils.write(VideoDataRaw, outputStream);
+
+        }
+    }
+
+    private static String humanTime(long millisec)
+    {
+        Timestamp timestamp = new Timestamp(millisec);
+        String date = timestamp.toString().split("\\.")[0];
+        return date;
+    }
+
+    private static long millisecondTime(String time) throws Exception
+    {
+        long milliTime = new java.text.SimpleDateFormat("MM/dd/yyyy~HH:mm:ss").parse(time).getTime();
```

Comment:
In general we would want to import java.text.SimpleDateFormat and then just refer to `SimpleDateFormat` here.